source/docs/software/sensors/accelerometers-software.rst

```diff
@@ -220,30 +220,22 @@ Most accelerometers legal for FRC use are quite noisy, and it is often a good id
 
         // Create a LinearDigitalFilter that will calculate a moving average of the measured X acceleration over the past 10 iterations of the main loop
 
-        LinearDigitalFilter xAccelFilter = LinearDigitalFilter.movingAverage(
-            new PIDSource() {
-                @Override
-                public PIDSourceType getPIDSourceType() {
-                    return PIDSourceType.kRate;
-                }
-
-                @Override
-                public double pidGet() {
-                    return accelerometer.getX();
-                }
-
-                @Override
-                public void setPIDSourceType(PIDSourceType pidSource) {
-                }
-            },
-            10);
+        LinearFilter xAccelFilter = LinearFilter.movingAverage(10);
 
         @Override
         public void robotPeriodic() {
             // Get the filtered X acceleration
-            double filteredXAccel = xAccelFilter.pidGet();
+            double filteredXAccel = xAccelFilter.calculate(accelerometer.getX());
         }
 
     .. code-tab:: c++
 
-        .. todo:: C++ example
+        frc::BuiltInAccelerometer accelerometer;
+
+        // Create a LinearDigitalFilter that will calculate a moving average of the measured X acceleration over the past 10 iterations of the main loop
```

Comment:
```suggestion
        // Create a LinearFilter that will calculate a moving average of the measured X acceleration over the past 10 iterations of the main loop
```

Reply:
Whoops, my bad for missing this. Fixed in 471f8ce

Reply:
My bad for missing this, fixed in 471f8ce